UnitsNet/UnitDefinitions/DcElectricPotential.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "DcElectricPotential",
```

Comment:
So from now on people will be forced to choose either ElectricPotentialAc or ElectricPotentialDc?

But there are many cases, in our software at least, where we don't care if its AC or DC...we just want to measure the Voltage.